app/ui/legacy/src/main/java/com/fsck/k9/ui/messageview/RecipientLayoutCreator.kt

```diff
@@ -30,7 +45,7 @@ internal class RecipientLayoutCreator(
 
         if (recipientNames.size == 1) {
             return RecipientLayoutData(
-                recipientNames = recipientsFormat.format(recipientNames.first()),
+                recipientNames = recipientNames.first().toDisplayRecipients(),
```

Comment:
I have struggle to figure out why the naming: `recipientNames` from `toDisplayRecipients()`
and later it's `displayRecipients` from `toDisplayRecipients()` asigned to `recipientNames`.

Wouldn't it be more consistent to alter `RecipientLayoutData` to be more clear why the one gets assigned to the other?

Reply:
I think the code now consistently uses `recipientList`. It could be a bit misleading because it's not a `List`. But then we're not typically using the type in a variable name. A `List<Recipient>` would typically be named `recipients`.

Another option I considered was `recipientLine`. However, the actual line that is displayed in the UI consists of the comma separated list of recipients and the number of additional recipients.

Reply:
Fine with me